signer/src/eth.rs

```diff
@@ -89,18 +97,10 @@ impl Keypair {
             .map_err(|_| Error::InvalidSeed)
     }
 
-    /// Obtain the [`ecdsa::PublicKey`] of this keypair.
-    pub fn public_key(&self) -> ecdsa::PublicKey {
-        self.0.public_key()
-    }
-
-    /// Obtains the public address of the account by taking the last 20 bytes
-    /// of the Keccak-256 hash of the public key.
-    pub fn account_id(&self) -> AccountId20 {
+    /// Obtain the [`eth::PublicKey`] of this keypair.
+    pub fn public_key(&self) -> PublicKey {
```

Comment:
Ah yeah ok; it's a bit different from sr25519 etc because the public key isn't directly the bytes we want for the acocuntId. I like this wrapper type though; I suppose it could just contain the `ecdsa::PublicKey` at this point, but I'm not too bothered either way :)